_examples/heatmap/demo/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"github.com/pterm/pterm"
+)
+
+func main() {
+	data := [][]float32{
+		{0.9, 0.2, -0.7, 0.4, -0.5, 0.6, -0.3, 0.8, -0.1, -1.0, 0.1, -0.8, 0.3},
+		{0.2, -0.7, -0.5, -0.3, -0.1, 0.1, 0.3, 0.5, 0.9, -0.9, -0.7, -0.5, -0.3},
+		{0.4, 0.4, -0.3, -1.0, 0.3, -0.2, -0.9, 0.5, -0.3, -1.0, 0.6, -0.2, -0.9},
+		{0.9, -0.5, -0.1, 0.3, 1, -0.7, -0.3, 0.1, 0.7, -0.9, -0.5, 0.2, 0.6},
+		{0.5, 0.6, 0.1, -0.2, -0.7, 0.8, 0.6, 0.1, -0.5, -0.7, 0.7, 0.3, 0.0},
+	}
+
+	headerData := pterm.HeatmapAxis{
+		XAxis: []string{"a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m"},
+		YAxis: []string{"1", "2", "3", "4", "5"},
+	}
+
+	table := pterm.DefaultHeatmap.WithAxisData(headerData).WithData(data)
```

Comment:
The Demo should be the "nicest" version of the printer. Could you enable RGB here?